tslog.c

```diff
@@ -42,14 +42,12 @@ int syslog_output(struct taskstats *t)
   return 1;
 }
 
-void help(char *s)
+void help(const char *s)
 {
-  printf("%s\n\n"
+  printf("\n\n"
```

Comment:
You have also removed the printing of `argv[0]` - which makes the argument unused then.
Besides I think that having that information in that output is useful.